src/Worker.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) 2021 Chiu Yen-Chen, Swen Sun-Yen, Wen Yong-Wei, Yuan Wei-Chen.
+All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file. See the AUTHORS file for names of contributors.
+ */
+#ifndef SHORTLINK_SRC_WORKER_H_
+#define SHORTLINK_SRC_WORKER_H_
+#include <string>
+#include "Worker.h"
+#include "Parser.h"
+
+namespace shortlink
+{
+	//constructor
+	Worker::Worker()
+	{
+		worker_status = WorkerStatus.kFree;
+		resp_data = "";
+		raw_request_data = "";
+	}
+	//start parse data
+	void Run(const std::string& raw_request_data)
+	{
+		this->raw_request_data = raw_request_data;
+		resp_data = Parser.Parse(this->raw_request_data);
+	}
+	//return parsed data to HTTP server class
+	const std::string Worker::SendResponse()
+	{
+		return raw_request_data;
```

Comment:
sendResponse() need to return parse data.
The parsing data is resp_data or raw_request_data?